scripts/future-auto-shutdown.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#set -x
+shopt -s nocasematch
+Rend_date='\033[0;31m'
```

Comment:
is this right?

Reply:
That was a "RED" var which looks to have got mixed up during a find and replace. We aren't actually using that so I have removed it altogether.